dmoj/monitor.py

```diff
@@ -70,7 +71,11 @@ def __init__(self):
                 self._refresher = None
 
             self._handler = SendProblemsHandler(self._refresher)
-            self._monitor = monitor = Observer()
+            if judgeenv.polling_observer:
+                self._monitor = monitor = PollingObserver()
+            else:
+                self._monitor = monitor = Observer()
```

Comment:
For style, change this to assign only `self._monitor` and change the only usage of `monitor` below to be `self._monitor` as well.